### PR TITLE
[Vectorize] Mention index name reuse + high-precision scoring trigger

### PR DESCRIPTION
**Pull request by netgusto**
### Summary

Updates the Vectorize doc to mention:
* index name reuse after deletion limitation
* how to trigger high precision scoring on queries

### CODE REVIEW

**Review by garvit-gupta — APPROVED** [review]

File: src/content/docs/vectorize/best-practices/create-indexes.mdx [hunk 1]
```diff
@@ -15,6 +15,8 @@ Creating an index requires three inputs:
 - The (fixed) [dimension size](#dimensions) of each vector, for example 384 or 1536.
 - The (fixed) [distance metric](#distance-metrics) to use for calculating vector similarity.
 
+An index cannot be created using the same name as an index previously deleted on your account.
```
Comment (sejoker) [inline src/content/docs/vectorize/best-practices/create-indexes.mdx]: we should note that we are working to address this limitation.

**Review by sejoker — APPROVED** [review]

File: src/content/docs/vectorize/best-practices/create-indexes.mdx (hunk 1 above)
Comment (netgusto) [reply src/content/docs/vectorize/best-practices/create-indexes.mdx]: Done.

File: src/content/docs/vectorize/best-practices/create-indexes.mdx [hunk 2]
```diff
@@ -103,4 +105,6 @@ Distance metrics are functions that determine how close vectors are from each ot
 
 Determining the similarity between vectors can be subjective based on how the machine-learning model that represents features in the resulting vector embeddings. For example, a score of `0.8511` when using a `cosine` metric means that two vectors are close in distance, but whether data they represent is _similar_ is a function of how well the model is able to represent the original content.
 
+When querying vectors, you can specify Vectorize to either use high-precision scoring, increasing the precision of the query matches scores as well as the accuracy of the query results, or use approximate scoring for faster response times. Using approximate scoring, returned scores will be an approximation of the real distance/similarity between your query and the returned vectors. See [Control over scoring precision and query accuracy](/vectorize/best-practices/query-vectors/#control-over-scoring-precision-and-query-accuracy)
```
Comment (Oxyjun) [inline src/content/docs/vectorize/best-practices/create-indexes.mdx]:
```suggestion
When querying vectors, you can specify Vectorize to use either:

- High-precision scoring, which increases the precision of the query matches scores as well as the accuracy of the query results.
- Approximate scoring for faster response times. Using approximate scoring, returned scores will be an approximation of the real distance/similarity between your query and the returned vectors. Refer to [Control over scoring precision and query accuracy](/vectorize/best-practices/query-vectors/#control-over-scoring-precision-and-query-accuracy).
```

File: src/content/docs/vectorize/best-practices/query-vectors.mdx [hunk 3]
```diff
@@ -71,6 +71,13 @@ This would return a set of matches resembling the following, based on the distan
 
 Refer to [Vectorize API](/vectorize/reference/client-api/) for additional examples.
 
+## Control over scoring precision and query accuracy
+
+When querying vectors, you can specify to either use high-precision scoring, increasing the precision of the query matches scores as well as the accuracy of the query results, or use approximate scoring for faster response times.
```
Comment (Oxyjun) [inline src/content/docs/vectorize/best-practices/query-vectors.mdx]:
```suggestion
When querying vectors, you can specify to either use high-precision scoring, thereby increasing the precision of the query matches scores as well as the accuracy of the query results, or use approximate scoring for faster response times.
```

**Review by Oxyjun — APPROVED** [review]

**Comment by netgusto** [comment]
@elithrar @pdwittig we need a :heavy_check_mark: if you have a sec

**Pull request merged**